Documentation/user-guides/configuring-cluster-monitoring.md

```diff
@@ -27,6 +27,8 @@ nodeExporter:
 kubeStateMetrics:
   baseImage: custom-registry.com/kube-state-metrics
   addonResizerBaseImage: custom-registry.com/addon-resizer
+openshiftStateMetrics:
```

Comment:
These changes come from the yaml file -> https://github.com/openshift/cluster-monitoring-operator/blob/9c97908591f4b4fc3b12251c4b6ab0d6289fdbc0/examples/user-guides/configuring-cluster-monitoring/custom-image-config.yaml

Reply:
@duritong friendly ping ^